docs/pages/operations-guide.md

```diff
@@ -371,361 +383,285 @@ Given the above configuration, let's assume `10.0.10.8` is being decommissioned
+1. Before updating the service itself, update its CLI subcommand to the new version:
+```bash
+$ dcos package uninstall --cli dse
+$ dcos package install --cli dse --package-version="1.1.6-5.0.7"
+```
+1. Once the CLI subcommand has been updated, simply call the update start command, passing in the version. For example, to update `dse` to version `1.1.6-5.0.7`:
+```bash
+$ dcos dse update start --package-version="1.1.6-5.0.7"
+```
 
-### Task logs
+If you are missing mandatory configuration parameters, the `update` command will return an error. To supply missing values, you can also provide an `options.json` file (see [Updating configuration](#updating-configuration) below):
+```bash
+$ dcos dse update start --options=options.json --package-version="1.1.6-5.0.7"
+```
 
-When the issue being diagnosed has to do with the service tasks, e.g. a given task is crash looping, the task logs will likely provide more information. The tasks being run as a part of a service are registered against a framework matching the service name. Therefore we should pick `<service-name>` from this list to view a list of tasks specific to that service.
+See [Advanced update actions](#advanced-update-actions) for commands you can use to inspect and manipulate an update after it has started.
 
-[<img src="img/ops-guide-framework-tasks-service.png" alt="listing of tasks running in a framework (Service tasks)" width="400"/>](img/ops-guide-framework-tasks-service.png)
+#### Updating configuration
 
-In the above list we see separate lists of Active and Completed tasks:
-- Active tasks are still running. These give a picture of the current activity of the service.
-- Completed tasks have exited for some reason, whether successfully or due to a failure. These give a picture of recent activity of the service. **Note:** Older completed tasks will be automatically garbage collected and their data may no longer be available here.
+The instructions below describe how you can update the configuration for a running DC/OS service.
 
-Either or both of these lists may be useful depending on the context. Click on the `Sandbox` link for one of these tasks and then start looking at sandbox content. Files named `stderr` and `stdout` hold logs produced both by the SDK Executor process (a small wrapper around the service task) as well as any logs produced by the task itself. These files are automatically paginated at 2MB increments, so older logs may also be examined until they are automatically pruned. For an example of this behavior, see the [scheduler sandbox](img/ops-guide-scheduler-sandbox.png) linked earlier.
+##### Preparing configuration
 
-[<img src="img/ops-guide-task-sandbox.png" alt="contents of a task sandbox" width="400"/>](img/ops-guide-task-sandbox.png)
+If you installed this service with Enterprise DC/OS 1.10, you can fetch the full configuration of a service (including any default values were applied during installation). For example, for `dse`:
```

Comment:
So this command isn't available in OSS? Or is it not available in previous DC/OS versions?

"including any default values that were...."